### DISCUSSION TITLE
autoApprove doesn't work on CodeCommit

### DISCUSSION THREAD

**Discussion by xfournet** [renovatebot/renovate#22337] [Request Help] [answered]
### How are you running Renovate?

Self-hosted

### If you're self-hosting Renovate, tell us what version of Renovate you run.

35.98.0

### If you're self-hosting Renovate, select which platform you are using.

AWS CodeCommit

### If you're self-hosting Renovate, tell us what version of the platform you run.

AWS CodeCommit

### Was this something which used to work for you, and then stopped?

I never saw this working

### Wanted end result.

I want to have some PR to be auto approved before being automerged by Renovate. Because BitBucket is configured to enforce to have at least one approval per PR in order to allow merge

### What you tried so far.

```
  "lockFileMaintenance": {
    "schedule": "on sunday after 5am and before 6am",
    "autoApprove": true,
    "automerge": true,
    "enabled": true
  }
````

### Relevant debug logs

<details><summary>Logs</summary>

```
<html>
<body>
<!--StartFragment-->

INFO: PR automerge failed (repository=xxxxxxxx/xxxx, branch=renovate-lock-file-maintenance)
--
136 | "pr": 3611

<!--EndFragment-->
</body>
</html>
```

</details>

**Comment by HonkingGoose** [comment]
I think you need to remove the `before 6am` part of the schedule. I  think Renovate does not have enough time to run.

**Comment by rarkins** [comment, marked as answer]
![image](https://github.com/renovatebot/renovate/assets/6311784/c74261c4-99eb-4790-8125-1d55b72f1862)
